spyder/widgets/panels/linenumber.py

```diff
@@ -178,14 +178,6 @@ def get_markers_margin(self):
         else:
             return 0
 
-    def update_(self, qrect, dy):
-        """Update line number area"""
-        if dy:
-            self.scroll(0, dy)
-        else:
-            self.update(0, qrect.y(),
-                        self.width(),
-                        qrect.height())
```

Comment:
It's implemented in the `PanelManager` class, It's enough to set `scrollable=True` in a `Panel` to make it scroll with the editor